tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -6710,7 +6711,7 @@ declare namespace ts.server.protocol {
          * Optional prefix to apply to possible completions.
          */
         prefix?: string;
-        triggerCharacter?: string;
+        triggerCharacter?: "." | "'" | '"' | "`" | "<";
```

Comment:
CompletionsTriggerCharacter ?

Reply:
This is from `protocol.ts` which I don't believe is supposed to reference types from outside itself?

Reply:
you can update the protocol generation script to copy type aliases, but that can be done in a separate PR.